frame/bridge/relay-authorities/src/lib.rs

```diff
@@ -231,19 +236,24 @@ decl_module! {
 					<Error<T, I>>::CandidateAE
 				);
 
+				// Max candidates can't be zero
 				if candidates.len() == T::MaxCandidates::get() {
-					ensure!(
-						stake >
-							candidates
-								.iter()
-								.map(|candidate| candidate.stake)
-								.max()
-								.unwrap_or(0.into()),
-						<Error<T, I>>::StakeIns
-					);
+					let mut minimum_stake = candidates[0].stake;
```

Comment:
This is to kick out the candidate with minimum stake?

Reply:
Yes. There's a max candidate count.